Content.Server/Dragon/DragonSystem.Rule.cs

```diff
@@ -31,10 +31,13 @@ protected override void Started(EntityUid uid, DragonRuleComponent component, Ga
     {
         base.Started(uid, component, gameRule, args);
 
-        if (!_station.Stations.Any())
+        var eligible = EntityQuery<StationEventEligibleComponent>().Select(x => x.Owner).ToList();
+
+        if (!eligible.Any())
             return;
 
-        var station = _random.Pick(_station.Stations);
+        var station = _random.Pick(eligible);
```

Comment:
There should probably be a helper function that finds a suitable station for events, rather than have each event code one itself.